tests/Pmi/Application/HpoApplicationTest.php

```diff
@@ -258,7 +258,7 @@ public function testAdminAutoselect()
         $client->followRedirects();
```

Comment:
I dropped this line from my other test in #801. The session variable will be set regardless, though.